doc-content/enterprise-only/installation/eap-users-create-proc.adoc

```diff
@@ -4,22 +4,33 @@
 Before you can log in to {CENTRAL} or {KIE_SERVER}, you must create users. This section shows you how to create a {CENTRAL} user with the `admin`,  `rest-all`, and `kie-server` roles and a {KIE_SERVER} user that has the `kie-server` role. For information about roles, see <<roles-users-con>>.
 
 .Prerequisites
-* {PRODUCT} is installed in the base directory of the {EAP} installation (`__EAP_HOME__`).
+* {PRODUCT} is installed in the base directory of the {EAP} installation (`_EAP_HOME_`).
 
 .Procedure
 . In a terminal application, navigate to the `__EAP_HOME__/bin` directory.
-. Create a user with the `admin` role that you will use to log in to {CENTRAL}. In the following command, replace `<username>` and   `<password>` with the user name and password of your choice.
+ifdef::PAM[]
+. Create a user with the `admin`, `rest-all`, and `kie-server` roles that you will use to log in to {CENTRAL}.
++
+NOTE: Users with the `admin` role are {CENTRAL} administrators. Users with `rest-all` role can access {CENTRAL} REST capabilities. Users with the `kie-server` role can access {KIE_SERVER} (KIE Server) REST capabilities. The `kie-server` role is mandatory for users to have access to *Manage and Track* views in {CENTRAL}.
```

Comment:
Shouldn't it be "**Manage** and **Track** views"  Meaning, two views, not one **Manage and Track** view. Right?